.golangci.yml

```diff
@@ -11,12 +19,10 @@ linters:
     - errcheck
     - errname
     - errorlint
-    - gci
```

Comment:
Why do you remove this linter for the import order?

Reply:
I didn't remove it, just moved it :)
![image](https://github.com/user-attachments/assets/7c207aad-d08d-429e-a316-3d18019ade7a)

https://golangci-lint.run/product/migration-guide

Reply:
Ok, didn't notice... Sorry for the noise!